project.clj

```diff
@@ -76,7 +76,7 @@
 
   :plugins [[lein-release "1.0.5" :exclusions [org.clojure/clojure]]]
 
-  :uberjar-name "puppet-server-release.jar"
+  :uberjar-name "puppetserver-release.jar"
```

Comment:
I don't *think* it will.  From a packaging perspective, the code on this PR seems to build a package that runs fine, even with the new uberjar name.  I did a local ezbake build off this branch to here - http://builds.puppetlabs.lan/puppetserver/2.4.0.master.SNAPSHOT.2016.04.20T0815/.  I installed that package on CentOS 7 and was able to start puppetserver and do an agent run against it.

I'm not sure that I'd consider the uberjar file name as "API", per se.  I think an upgrade would remove the old uberjar name and replace with this new one.

@haus - what do you think about this?  Would this change cause you any concern?